nvtabular/inference/triton/__init__.py

```diff
@@ -463,6 +472,13 @@ def get_column_types(path):
     return json.load(open(os.path.join(path, "column_types.json")))
 
 
+def get_slot_sizes(path):
+    if not path.exists(path):
+        return json.load(open(os.path.join(path, "slot_size_array.json")))
+    else:
+        return None
```

Comment:
The only caller of this function seems to be in model_hugectr.py - maybe we should move there?

Also the code in model_hugectr.py doesn't handle this function returning None - maybe just remove the check to see if the file exists?

Reply:
Updated this. And, moved it into model_hugectr.py